### PR TITLE
[GH-37] Implement slash command autocomplete

### PR DESCRIPTION
**Pull request by GodlikePenguin**
#### Summary

This commit adds autocomplete prompts for all of the subcommands offered by the plugin.
Adds a new `join` subcommand which is an alias for `/webex <@username>/<room id>. This was required as the Autocomplete library does not allow for a command to have both subcommands and arguments.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-plugin-webex/issues/37

### CODE REVIEW

**Comment by hanzei** [comment]
@GodlikePenguin Could you please run `go mod tidy` and commit the changes in `go.sum` to fix CI?

File: server/command.go [hunk 1]
```diff
@@ -81,9 +82,36 @@ func getCommand() *model.Command {
...
+	webexAutocomplete.AddCommand(start)
+
+	room := model.NewAutocompleteData("room", "<room id>", "Sets your personal Meeting Room ID")
+	room.AddTextArgument("Webex meeting room ID", "<room id>", "")
+	webexAutocomplete.AddCommand(room)
+
+	roomReset := model.NewAutocompleteData("room-reset", "", "Removes your room setting")
+	webexAutocomplete.AddCommand(roomReset)
+
+	join := model.NewAutocompleteData("join", "<room id>/<@username>", "Shares a link to a Webex meeting from a room id or a Mattermost username")
```
Comment (cpoile) [inline server/command.go:108]:
```suggestion
	join := model.NewAutocompleteData("join", "<room id>/<@username>", "Shares a link to a Webex meeting in <room id> or in <@username>'s meeting room")
```

**Review by cpoile — CHANGES_REQUESTED** [review]
Thank you @GodlikePenguin, this is great!
One suggestion for the help text, otherwise fantastic.

**Review by iomodo — APPROVED** [review]
LGTM! Thanks @GodlikePenguin

**Review by cpoile — APPROVED** [review]
Nice, thanks @GodlikePenguin !

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Autocomplete is working as expected
- Tested on desktop and browser
- Confirmed join command is functional

LGTM! 

Huge thanks @GodlikePenguin for this enhancement!

**Pull request merged**